sdk/storage/blobs/inc/az_storage_blobs.h

```diff
@@ -53,28 +80,39 @@ typedef struct
   az_span option;
```

Comment:
Should this field be internal? What does it mean?

Reply:
Moved to internal

Reply:
nit: also rename it to unused?